.github/workflows/dev-image-update.yaml

```diff
@@ -98,3 +94,29 @@ jobs:
       gcr_tag: ${{ needs.update_image.outputs.ui_image_tag }}
       source_gcr_url: 'gcr.io/broad-jade-dev/jade-data-repo-ui'
       target_gcr_url: 'gcr.io/datarepo-public-gcr/jade-data-repo-ui'
+
+  report-to-sherlock:
+    name: Report App Version to DevOps
+    uses: broadinstitute/sherlock/.github/workflows/client-report-app-version.yaml@main
+    needs: [update_image, cherry_pick_image_to_production_gcr]
+    with:
+      new-version: ${{ needs.update_image.outputs.ui_image_tag }}
+      chart-name: datarepo-ui
+    permissions:
+      contents: read
+      id-token: write
```

Comment:
It makes sense to me to add the report to sherlock, but I'm surprised we're ready to pull the helm_tag_bump action already. Would it help to report to both places for an interim time period?